spiffworkflow-backend/docs/pycharm.md

```diff
@@ -0,0 +1,54 @@
+# PyCharm Setup Instructions
+
+## Pre-Requisites
+* Before starting, be sure you can run bin/run_server_locally script successfully.
+This will prove that your libraries are properly configured.
+* Checkout a git repository with sample process models such as our [Sample Process Models](https://github.com/sartography/sample-process-models)
+
+
+## Step 1 - Python Interpreter
+In the settings, ensure that you have configured your Python interpreter to use the poetry environment you have created.
+
+## Step 2 - Configuration Settings
+Create a config.py file in the src/instance directory, with the following base settings
```

Comment:
not required as a part of this change, i wonder if we should put all/most of these in config/local_development.py.

even the secrets i'm not super worried about if it's labelled local development.